soperator/installations/example/variables.tf

```
@@ -332,7 +333,8 @@ variable "slurm_nodeset_system" {
   })
   nullable = false
   default = {
-    size = 1
+    min_size = 3
+    max_size = 9
     resource = {
       platform = "cpu-e2"
```

```suggestion
      platform = "cpu-d3 "
```
I think we want to default to this now.